examples/tutorials/advanced/grid_equalization.py

```diff
@@ -0,0 +1,221 @@
+        fig.grdimage(grid=linear, projection="M?", frame="WSne", cmap=True)
+    # Plot a histogram showing the linear z-value distribution
+    with fig.set_panel(panel=1):
+        fig.histogram(
+            data=linear_dist,
+            projection="X?",
+            region=[0, divisions, 0, 40],
+            series=[0, divisions, 1],
+            frame="wnSE",
+            cmap=True,
+            histtype=1,
+            pen="1p,black",
+        )
+        fig.colorbar(position="JMR+o1c/0c+w3c/0.3c", frame=True)
+fig.show()
+
+###############################################################################
+# Transform grid based on a normal distribution
+# ---------------------------------------------
+# The ``gaussian`` parameter of :meth:`pygmt.grdhisteq.equalize_grid` can be
+# used to transform the z-values relative to their position in a normal
+# distribution rather than a linear distribution. In this case, the output
+# data are continuous rather than discrete.
+
+normal = pygmt.grdhisteq.equalize_grid(grid=grid, gaussian=True)
+normal_dist = pygmt.grd2xyz(grid=normal, output_type="pandas")["z"]
+
+###############################################################################
+# Plot the normally distributed data
+# ----------------------------------
+# Here we create a map showing the grid that has been transformed to have
+# a normal distribution and a histogram of the data values.
+
+# Create an instance of the Figure class
+fig = pygmt.Figure()
+# Define figure configuration
+pygmt.config(FORMAT_GEO_MAP="ddd.x", MAP_FRAME_TYPE="plain")
+# Define the colormap for the figure
+pygmt.makecpt(series=[-4.5, 4.5], cmap="vik")
+# Setup subplots with two panels
+with fig.subplot(
+    nrows=1, ncols=2, figsize=("13.5c", "4c"), title="Normal distribution"
+):
+    # Plot the grid with a normal distribution in the first panel
+    with fig.set_panel(panel=0):
+        fig.grdimage(grid=normal, projection="M?", frame="WSne", cmap=True)
+    # Plot a histogram showing the normal z-value distribution
+    with fig.set_panel(panel=1):
+        fig.histogram(
+            data=normal_dist,
+            projection="X?",
+            region=[-4.5, 4.5, 0, 20],
+            series=[-4.5, 4.5, 0.2],
+            frame="wnSE",
+            cmap=True,
+            histtype=1,
+            pen="1p,black",
+        )
+        fig.colorbar(position="JMR+o1c/0c+w3c/0.3c", frame=True)
+fig.show()
+
+###############################################################################
+# Equalize grid based on a quadratic distribution
+# -----------------------------------------------
+# The ``quadratic`` parameter of :meth:`pygmt.grdhisteq.equalize_grid` can be
+# used to transform the z-values relative to their position in a quadratic
+# distribution rather than a linear distribution. Here, we equalize the grid
+# into nine divisions based on a quadratic distribution and produce a
+# pandas.Series with the z-values for the new grid.
```

Comment:
```suggestion
# pandas. Series with the z-values for the new grid.
```

Reply:
Why should there be a space here?

Reply:
Sorry, I meant to use :class:`pandas.Series` here. Updated the suggestion.